src/app/d2-loadout-builder/generated-sets/GeneratedSetButtons.tsx

```diff
@@ -51,6 +51,7 @@ export default function GeneratedSetButtons({
  */
 function createLoadout(classType: DimStore['class'], set: ArmorSet): Loadout {
   const loadout = newLoadout(
+    // TODO: Move ...set.stats out of t() in breaks i18next-scanner
     t('Loadouts.Generated', { ...set.stats, tier: _.sum(Object.values(set.stats)) }),
```

Comment:
@bhollis is there any way to move this spread operator into (Object.values()) ?

Reply:
You could always assign the options to a local var, maybe that'll prevent it from getting confused.

Reply:
disregard. set it in `data: {}`